buildspec_canary.yml

```diff
@@ -0,0 +1,14 @@
+version: 0.2
```

Comment:
Don't forget to change https://tiny.amazon.com/ne2o910k/IsenLink to use this new buildspec for canary

Reply:
this is a CM step!